docs/tables/snowflake_database_grant.md

```diff
@@ -0,0 +1,36 @@
+# Table: snowflake_database_grant
+
+Lists privileges that have been granted on the database.
+
+**Note** This table requires an '=' qualifier for `database` columns.
```

Comment:
What `database` columns are you referring to here, `snowflake_database` table columns, or the `database` column in this table?